src/FiltrationGroupComponent/FiltrationGroupComponent.tsx

```diff
@@ -8,7 +8,16 @@ import "./FiltrationGroupComponent.scss";
 
 type Props = StateProps & CallbackProps;
 
+interface StateDefaultProps {
+	horizontalBracket: [];
+	verticalBracket: boolean;
+}
+
 export class FiltrationGroupComponent extends React.Component<Props> {
+	public state: StateDefaultProps = {
```

Comment:
Почему, кстати, Props в имени?
Может так?
```suggestion
	public state: State = {
```